src/userobjects/MyTRIMRasterizer.C

```diff
@@ -541,6 +548,13 @@ MyTRIMRasterizer::periodicPoint(const Point & pos) const
   return p;
 }
 
+bool
+MyTRIMRasterizer::isTrackedSpecies(unsigned int atomic_number, Real mass_number) const
+{
+  mooseAssert(_pka_generators[0], "PKA generator is not set.");
+  return _pka_generators[0]->ionTag(_pka_parameters, atomic_number, mass_number) != -1;
```

Comment:
This works for now, but I think we can improve the design. `ionTag` should be a member of the Rasterizer and we should hand the PKAGenerators a `const MyTRIMRasterizer &` instead of the data package.